docs/standard/native-interop/customize-struct-marshaling.md

```diff
@@ -331,7 +330,7 @@ struct Currency
 };
 ```
 
-## Marshaling `System.Object`s
+## Marshal System.Objects
```

Comment:
By that, reader will assume that class name is "Objects" not Object". That change will also make it localized.

I think back ticks should exist as it was, and only change Marshaling to Marshal.

Reply:
I changed it to "Marshal System.Object fields". I would think the machine translators know not to translate System.xyz strings, and we're not supposed to put formatting in headings.